pkg/storkctl/action.go

```diff
@@ -14,20 +14,23 @@ import (
 )
```

Comment:
Once the CLI is finalized, lets plan to add UTs to the action commands like we have for others.

Reply:
Created a ticket to track it https://portworx.atlassian.net/browse/PWX-30833
I will add the tests once the CLI is finalized